client/src/components/PublicPages/AuditPlanner/IntegerInput.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { InputGroup } from '@blueprintjs/core'
+
+interface IProps {
+  ariaLabel?: string
+  name: string
+  onChange: (newValue: number) => void
+  value: number
+}
+
+const IntegerInput: React.FC<IProps> = ({
+  ariaLabel,
+  name,
+  onChange,
+  value,
+}) => {
+  return (
+    <InputGroup
+      aria-label={ariaLabel}
+      name={name}
+      onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+        const newValue = Math.round(Number(e.target.value))
+        onChange(newValue)
+      }}
+      placeholder="0"
+      type="number"
+      value={value === 0 ? '' : `${value}`}
```

Comment:
Distinguishing an explicitly entered `0` from no input is an interesting edge case. Properly handling this would require making the `value` prop optional, e.g. `value?: number`, and updating consumers / consumer state to distinguish `0` and `undefined` - the latter piece would add some complexity that didn't feel worth it for the rare (I think) case that someone needs to enter 0 votes for a candidate

Right now, if you try entering an explicit `0`, the input will ignore your key press and continue displaying the placeholder `0`. The `0` will still be properly recorded under the hood